content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -480,6 +480,7 @@ To uninstall Istio, run the following command:
 {{< text bash >}}
 $ kubectl delete -n istio-system -f @samples/multicluster/expose-istiod.yaml@
 $ istioctl manifest generate | kubectl delete -f -
+$ istioctl uninstall -y --purge
```

Comment:
If you add this call to `istioctl uninstall`, you should be able to remove the previous line (`istioctl manifest generate | kubectl delete`). You don't need both?

Reply:
Removed